docs/UserGuide.md

```diff
@@ -241,15 +273,34 @@ This also means that you can use `t/` to remove all tags from a dentist using th
 
 ## Part 2 - Patient Features
 
-### Adding a patient: `add-patient`
-
+### Adding a patient : `add-patient`
 Adds a patient to the list of patients in ToothTracker. This is helpful when:
-- You are using ToothTracker for the first time and you have to add your patients' particulars.
-- A new patient has joined your clinic.
 
-Format: `add-patient n/NAME p/PHONE b/BIRTHDATE g/GENDER [r/REMARK] [tr/TREATMENT] [e/EMAIL] [h/ADDRESS] ADDRESS] [t/TAG] `
+* You are using ToothTracker for the first time, and you have to add your patients' particulars.
+* A new patient has joined your clinic.
 
-Examples:
+**Format:** `add-patient n/NAME p/PHONE b/BIRTHDATE g/GENDER [r/REMARK] [tr/TREATMENT] [e/EMAIL] [h/ADDRESS] [t/TAG]…​`
+
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+A patient can have any number of <code>TAG</code>s (including zero tags).
+
+<code>EMAIL</code>, <code>ADDRESS</code>, <code>REMARK</code> and <code>TREATMENT</code> are optional to be put.
+You may use the `edit-patient` command to update them in the future.
+</div>
+
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes about adding a patient without remark and treatment:**<br>
+ToothTracker will put the <code>REMARK</code> and <code>TREATMENT</code> as `NIL` by default.
+
+<div markdown="block" class="alert alert-info">
+**:information_source: Notes about adding a patient without email and address:**<br>
+ToothTracker will put the <code>EMAIL</code> as "NoEmailProvided@ToBeAdded.com"
+and put the <code>ADDRESS</code> as "No Address Provided." by default.
+
+If you wish to update them at a later time, you can use <code>edit-patient</code> to edit them.
+</div>
+
+**Examples:**
 
 * `add-patient n/John p/90676622 b/06-06-1998 g/M r/Allergic to Peanuts tr/Braces e/johntan@gmail.com h/60 Harvey Avenue t/Urgent` <br>
```

Comment:
I think it should requesting for Braces treatment, not cleaning treatment.

Reply:
Also, should the explanation of the example be written by following the order of the patient format?